diff --git a/Change max length of the graph name/items.inc.php b/Change max length of the graph name/items.inc.php
index 4d55a6a..6a507cd 100644
--- a/Change max length of the graph name/items.inc.php	
+++ b/Change max length of the graph name/items.inc.php	
@@ -1393,8 +1393,8 @@ function formatHistoryValue($value, array $item, $trim = true) {
 
 		case ITEM_VALUE_TYPE_TEXT:
 		case ITEM_VALUE_TYPE_LOG:
-			if ($trim && mb_strlen($value) > 20) {
-				$value = mb_substr($value, 0, 20).'...';
+			if ($trim && mb_strlen($value) > 60) {
+				$value = mb_substr($value, 0, 60).'...';
 			}
 
 			if ($mapping !== false) {
